src/resources/resources.controller.ts

```diff
@@ -25,6 +42,27 @@ export class ResourcesController {
         summary:
             "Adds a URL with title to the team's resources, addedBy: teamMemberId (int)",
     })
+    @ApiResponse({
```

Comment:
Can you review this? There is an undocumented 401 error here.
![Screenshot 2024-01-12 at 3 42 54 PM](https://github.com/chingu-x/chingu-dashboard-be/assets/80556079/bcadc3d7-9eee-4245-bbe0-cfecc8462b33)